libass/ass_render.c

```diff
@@ -3428,6 +3428,8 @@ ASS_Image *ass_render_frame(ASS_Renderer *priv, ASS_Track *track,
     ass_frame_unref(priv->prev_images_root);
     priv->prev_images_root = NULL;
 
+    ass_prune_history(track, now, track->parser_priv->gc_threshold);
```

Comment:
Directly discarding anything prior to “now” breaks even simple, short rewinds which might be too harsh for most. Perhaps auto-gc should also take a time as an offset from “now”